### PR TITLE
Add OSIP 2017/18

### PR DESCRIPTION
**Pull request by arch0345**
This PR adds the newest [Oregon Statewide Imagery Program](https://spatialdata.oregonexplorer.info/geoportal/imagery) (OSIP) Imagery, which is currently split between Eastern (2017) and Western (2018) Oregon. This imagery is the best imagery outside of urban areas in Oregon and was released by the Oregon Geospatial Enterprises Office, which [claims](https://www.oregon.gov/geo/Pages/faq.aspx) that their data is in the public domain. I added the State of Oregon privacy policy URL since the imagery is hosted by the State of Oregon Enterprise Information Services.

### CODE REVIEW

**Comment by cicku** [comment]
Please update description “Newest OSIP Imagery for Eastern Oregon” with certain years.

**Comment by cicku** [comment]
This is so interesting. I asked them in the last year and they told me something different.

Let me check it.

**Comment by arch0345** [comment]
> This is so interesting. I asked them in the last year and they told me something different.
> 
> Let me check it.

Hmmm maybe OSIP is licensed differently than other Oregon GEO data? I'll email them about this (unless you've done so already).

**Comment by cicku** [comment]
Yes please go ahead. I will attach my email history here. They did not mention that the imagery has license.

OSIP

File: sources/north-america/us/or/OSIP_2017.geojson [hunk 1]
```diff
@@ -0,0 +1,556 @@
+{
+    "type": "Feature",
+    "properties": {
+        "license_url": "https://www.oregon.gov/geo/Pages/faq.aspx",
+	"privacy_policy_url": "https://www.oregon.gov/pages/terms-and-conditions.aspx",
+        "id": "OSIP_2017",
```
Comment (1ec5) [inline sources/north-america/us/or/OSIP_2017.geojson:6]: Oh, another OSIP! Good thing #1431 spells out Ohio’s OSIP. 😉 I didn’t consider the potential for conflicts in the IDs, but I’ll have to keep this in mind if we add the year-based layers alongside the resolution-based ones.

**Comment by arch0345** [comment]
Update: just got a reply from the head of the Imagery Framework Implementation Team (see image below). I'll let you know when they reply again

**Comment by Asteliks** [comment]
@arch0345 might you have an update?

**Comment by arch0345** [comment]
> @arch0345 might you have an update?

I emailed them a few months ago asking if they had an update on this:

image

I haven't gotten a response since then.

Although NAIP 2020 is more recent, [its resolution isn't as good as OSIP](https://www.oregon.gov/geo/Pages/imagery_data.aspx). It looks like NAIP 2022 will have imagery at a similar resolution, however, I think that it'll still be good to have OSIP in the index as another source to compare to.

**Comment by Asteliks** [comment]
@arch0345 okey in this case what do we want to do? Without solving the copyright issue we can not merge this source. Should we give this PR let’s say 2 more weeks and if nothing moves forward than we will have it closed?

**Comment by arch0345** [comment]
Still haven't gotten a response, closing for now

**Pull request closed without merging**